packages/scrollable-image-ui/README.md

```diff
@@ -0,0 +1,19 @@
+# scrollable-image-ui
+
+## What is this
+
+An interface used for generating scrollable image embedded code which is served by google cloud function.
+
+## How to develop this
+
+```bash
+# Run development server
+make dev
+```
+
+## How to build this
+
+```bash
+# Build public/ files which will be deployed to google cloud function
+make build
+```
```

Comment:
README.md should be modified since this package is not to deal with google cloud functions.

Reply:
updated.